src/commands/new.ts

```diff
@@ -126,6 +127,7 @@ export default {
 
     const yname = !!options.y || !!options.yes
     const useDefault = (option: unknown) => yname && option === undefined
+    const boolFlag = (option: unknown) => (option !== undefined ? bool(option) : undefined)
```

Comment:
Actually, let's add this logic into your `bool` function and just call the exported function `boolFlag`. Check for `undefined` first with

```
if (value === undefined) return value
```

Reply:
Extracted in this commit: https://github.com/infinitered/ignite/pull/2062/commits/010b913d12df0738c6c9a2faacf502bea814981d